Flora-Fauna-Frontend/src/components/UserCards/UserCards.jsx

```diff
@@ -1,11 +1,55 @@
 import "./UserCards.css";
+import { useState } from "react";
+import ModalPopup from "../ModalPopup/ModalPopup";
+import { toBase64 } from "../UserFeed/UserFeed";
 
 export default function UserCards(props){
+
+  const { source, title, desc, post } = props
+
+  const [visible, setVisible] = useState(false);
+  const [modalContent, setModalContent] = useState([]);
+  const modalHandler = () => setVisible(true);
+  
+  const closeHandler = () => {
+    setVisible(false);
+    console.log("closed");
```

Comment:
Remove/Comment out

Reply:
When I tried it, the modals would only open and close one time, then never again.

Reply:
That doesn't sound write... just delete the console.log

Reply:
Yes, I removed it